public/app/plugins/panel/dashlist/styles.ts

```diff
@@ -2,52 +2,55 @@ import { css } from '@emotion/css';
 
 import { GrafanaTheme2 } from '@grafana/data';
 
-export const getStyles = (theme: GrafanaTheme2) => ({
-  dashlistSectionHeader: css`
-    padding: ${theme.spacing(0.25, 1)};
-    margin-right: ${theme.spacing(1)};
-  `,
-  dashlistSection: css`
-    margin-bottom: ${theme.spacing(2)};
-    padding-top: 3px;
-  `,
-  dashlistLink: css`
-    display: flex;
-    cursor: pointer;
-    border-bottom: 1px solid ${theme.colors.border.weak};
-    margin-right: ${theme.spacing(1)};
-    padding: ${theme.spacing(1)};
-    align-items: center;
+export const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    dashlistSectionHeader: css({
+      padding: theme.spacing(0.25, 1),
+      marginRight: theme.spacing(1),
+    }),
+    dashlistSection: css({
+      marginBottom: theme.spacing(2),
+      paddingTop: theme.spacing(0.5),
+    }),
+    dashlistLink: css({
+      display: 'flex',
+      cursor: 'pointer',
+      borderBottom: `1px solid ${theme.colors.border.weak}`,
+      margin: theme.spacing(1),
+      padding: theme.spacing(1),
+      alignItems: 'center',
 
-    &:hover {
-      a {
-        color: ${theme.colors.text.link};
-        text-decoration: underline;
-      }
-    }
-  `,
-  dashlistFolder: css`
-    color: ${theme.colors.text.secondary};
-    font-size: ${theme.typography.bodySmall.fontSize};
-    line-height: ${theme.typography.body.lineHeight};
-  `,
-  dashlistTitle: css`
-    &::after {
-      position: absolute;
-      content: '';
-      left: 0;
-      top: 0;
-      bottom: 0;
-      right: 0;
-    }
-  `,
-  dashlistLinkBody: css`
-    flex-grow: 1;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  `,
-  dashlistItem: css`
-    position: relative;
-    list-style: none;
-  `,
-});
+      '&:hover': {
+        a: {
+          color: theme.colors.text.link,
+          textDecoration: 'underline',
+        },
+      },
+    }),
+    dashlistFolder: css({
+      color: theme.colors.text.secondary,
+      fontSize: theme.typography.bodySmall.fontSize,
+      lineHeight: theme.typography.body.lineHeight,
+    }),
+    dashlistTitle: css({
+      '&::after': {
+        position: 'absolute',
+        content: '""',
+        left: 0,
+        top: 0,
+        bottom: 0,
+        right: 0,
+      },
+    }),
+    dashlistLinkBody: css({
+      flexGrow: 1,
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      padding: theme.spacing(0.25, 0.5),
```

Comment:
think a better fix here would be to remove the `overflow` and `textOverflow` props (as well as the `padding`) since they're not doing anything anyway 😄